include/signature.h

```diff
@@ -73,13 +73,15 @@ GBytes *cms_sign_file(const gchar *filename, const gchar *certfile, const gchar
  *
  * @param content content to verify against signature
  * @param sig signature used to verify
+ * @param capath path to certificate file to use for verification
+ * @param cadir path to certificate dir to use for verification
```

Comment:
Hmm, maybe we should look at passing a preparing a store for use instead of adding more parameters?

Reply:
Probably the better (more generic) option, yes.

Reply:
Next round... After a lot of refactoring and rework in test/signature.c, here is not the approach for passing a store directly.